allay/parser.py

```diff
@@ -327,15 +327,16 @@ def parse_standalone(self, stream: TokenStream) -> dict:
                         raise InvalidSyntax(f"Unknown template '{q}'")
 
                     args = self.parse_template_args(stream)
-                    if args:
-                        # Turn the template into a string so we can use replace on it
-                        template_value = json.dumps(self.templates[q])
-                        # Replace all the tokens
-                        for index, arg in enumerate(args):
-                            template_value = template_value.replace(f"%{index}", arg)
-
-                        # Convert it back into a dictionary
-                        standalone_contents = json.loads(template_value)
+
```

Comment:
Good catch. Didn't notice that `self.parse_template_args` always returns a list=. So now templates without args will work.